utils/backend.py

```diff
@@ -34,12 +34,15 @@ def do_post(url, data, headers):
     retry = 1
     while retry <= MAX_RETRIES:
         r = requests.post(url, data=data, headers=headers)
-        # FIXME: should use the correct code for no respose here?
-        if r.status_code == 200:
+
+        # 413 means payload too large, it happens when we try to submit too
+        # much data; in this situation there's not point in retrying
+        if r.status_code == 200 or r.status_code == 413:
+            if r.status_code == 413:
+                print("Aborting because payload was too large")
```

Comment:
We could submit only a part of it with an info that it is too large :)

Reply:
Or just a comment that the change is too large.

Reply:
Done, the HTTP error is sent as comment, if possible!